chrome/js/background.js

```diff
@@ -13,4 +12,13 @@ chrome.runtime.onInstalled.addListener(function () {
       }
     ]);
   });
+  chrome.contextMenus.removeAll();
+  chrome.contextMenus.create({
+    id: "enhance",
+    title: "CSI Enhance",
+    contexts: ["selection"],
+  });
+  chrome.contextMenus.onClicked.addListener((info, tabs) => {
```

Comment:
Any thoughts on how this should be structured filewise? should this exist in background or maybe it's own file?